src/sentry/mail/actions.py

```diff
@@ -55,7 +56,7 @@ def clean(self):
         if targetType != self.teamValue and targetType != self.memberValue:
             return
 
-        if targetIdentifier is None:
+        if targetIdentifier is None or targetIdentifier == "":
```

Comment:
Leaving the same comment here as the other pr: This could just be `not targetIdentifier`

Reply:
Oh LOL guess Chris beat me to it